browser-extension/html/tournesol-iframe.css

```diff
@@ -0,0 +1,34 @@
+html, body {
+    height: 100%;
+}
+
+body {
+    margin: 0;
+    overflow: hidden;
+}
+
+.main {
+    width: 100%;
+}
+
+.heading {
+    height: 22px;
```

Comment:
Is it necessary to define an explicit "height" here?  
It breaks the layout in smaller screens:
![image](https://user-images.githubusercontent.com/4726554/151176002-9227b890-84ce-4516-ae19-e9675b23a62f.png)

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/492/commits/09919b277c2edf4c51654004715159c8c9f955e6